app/views/api/v1/restaurants/index.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.data do
+  json.restaurants @restaurants do |restaurant|
+    json.restaurant_name restaurant.name
+    json.restaurant_address restaurant.address
+    json.restaurant_description restaurant.description
+    json.restaurant_latitude restaurant.latitude
+    json.restaurant_longitude restaurant.longitude
+    json.restaurant_categories @categories do |category|
```

Comment:
This is okay but why not just use `restaurant.restaurant_categories` instead of `@categories`? That would save you a call in the controller if you make use of Eager Loading i.e. `Restaurant.all.includes(:restaurant_categories)` But even without it, I can't really see what the point of fetching all `RestaurantCategory` instances from the database.